test/tools/ossfuzz/protoToYul.h

```diff
@@ -295,12 +309,16 @@ class ProtoConverter
 	}
 
 	std::ostringstream m_output;
-	/// Variables in current scope
-	std::stack<std::vector<std::string>> m_scopeVars;
+	/// Variables in all function definitions
+	std::vector<std::vector<std::vector<std::string>>> m_funcVars;
+	/// Variables in current function definition
+	std::vector<std::string const*> m_currentFuncVars;
+	/// Variables in global scope
+	std::vector<std::string const*> m_currentGlobalVars;
 	/// Functions in current scope
-	std::stack<std::vector<std::string>> m_scopeFuncs;
+	std::vector<std::vector<std::string>> m_scopeFuncs;
```

Comment:
Why 2d vector instead of 1d?

Reply:
2d because it is a vector of scopes in which each scope is a vector of function names.